pkg/api/routes.go

```diff
@@ -1565,6 +1593,77 @@ func (rh *RouteHandler) ListExtensions(w http.ResponseWriter, r *http.Request) {
 	WriteJSON(w, http.StatusOK, extensionList)
 }
 
+// Openid CodeExchange callback.
+func (rh *RouteHandler) UserInfoLogic() rp.CodeExchangeUserinfoCallback {
```

Comment:
This should be the only function where we are using `email` instead of `identity`.